com.ibm.streamsx.topology/info.xml

```diff
@@ -41,7 +41,7 @@ Applications use Python code to process tuples and tuples are Python objects.
 See the documentation under [namespace:com.ibm.streamsx.topology.python|com.ibm.streamsx.topology.python] for details.
 
     </description>
-    <version>1.2.1.experimental</version>
+    <version>1.2.1.alpha</version>
```

Comment:
Can you bump this to 1.2.5.

Since this will be a new release we need to have a different version number to the previous one.